catalyst_voices/lib/widgets/headers/segment_header.dart

```diff
@@ -38,30 +38,33 @@ class SegmentHeader extends StatelessWidget {
     final textStyle = (theme.textTheme.titleMedium ?? TextStyle())
         .copyWith(color: foregroundColor.resolve(_states));
 
-    return IconButtonTheme(
-      data: IconButtonThemeData(style: iconButtonStyle),
-      child: TextButtonTheme(
-        data: TextButtonThemeData(style: textButtonStyle),
-        child: AnimatedDefaultTextStyle(
-          duration: kThemeChangeDuration,
-          style: textStyle,
-          maxLines: 1,
-          overflow: TextOverflow.ellipsis,
-          child: AnimatedContainer(
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 4),
```

Comment:
so this is more like `margin`, imo it should not be here but where this widget is used